script.js

```diff
@@ -0,0 +1,193 @@
+  {
+    name: 'Post Stories',
+    description: 'Experimental content creation feature that allows users to add to an existing story over the course of a day without spamming their friends.',
+    mobileImage: './images/Snapshoot Portfolio (1).svg',
+    desktopImage: './images/Snapshoot Portfolio 2.svg',
+    technologies: [
+      'HTML',
+      'Ruby on Rails',
+      'CSS',
+      'JavaScript',
+    ],
+    liveVersionLink: '',
+    SourceLink: '',
+    company: 'FACEBOOK',
+    year: '2015',
+    stack: 'Full Stack Dev',
+  },
+  {
+    name: 'Facebook 360',
+    description: 'Exploring the future of media in Facebook\'s first Virtual Reality app; a place to discover and enjoy 360 photos and videos on Gear VR.',
+    mobileImage: './images/Snapshoot Portfolio (2).svg',
+    desktopImage: './images/Snapshoot Portfolio 3.svg',
+    technologies: [
+      'HTML',
+      'Ruby on Rails',
+      'CSS',
+      'JavaScript',
+    ],
+    liveVersionLink: '',
+    SourceLink: '',
+    company: 'FACEBOOK',
+    year: '2015',
+    stack: 'Full Stack Dev',
+  },
+  {
+    name: 'Uber Navigation',
+    description: 'A smart assistant to make driving more safe, efficient, and fun by unlocking your most expensive computer: your car.',
+    mobileImage: './images/Snapshoot Portfolio (3).svg',
+    desktopImage: './images/Snapshoot Portfolio 4.svg',
+    technologies: [
+      'HTML',
+      'Ruby on Rails',
+      'CSS',
+      'JavaScript',
+    ],
+    liveVersionLink: '',
+    SourceLink: '',
+    company: 'Uber',
+    year: '2018',
+    stack: 'Lead Developer',
+  },
+];
+
+const popup = document.querySelector('.overlay');
+const closePopup = document.querySelector('.closebtn');
+const subDesc = document.querySelector('.popsubdesc');
+const workImg = document.querySelector('.popMimg');
+const deskImg = document.querySelector('.popDimg');
+const popupTitle = document.querySelector('.poptitle');
+const popupContent = document.querySelector('.pop-work');
+const skillsDiv = document.querySelector('.poplangs');
+const workDesc = document.querySelector('.popdesc');
+const liveBtn = document.querySelector('.lbtn');
+const sourceBtn = document.querySelector('.sbtn');
+
+const generatePopup = (index) => {
+  skillsDiv.innerHTML = '';
+  popupTitle.innerText = projects[index].name;
+  workImg.src = projects[index].mobileImage;
+  deskImg.src = projects[index].desktopImage;
+  workDesc.innerText = projects[index].description;
+  projects[index].technologies.forEach((skill) => {
+    const li = document.createElement('li');
+    li.innerText = skill;
+    skillsDiv.appendChild(li);
+  });
+  subDesc.innerHTML = `
+  <p class="descA">${projects[index].company}</p>
+  <img src="./images/Counter.svg" alt="counter">
+  <p class="grey">${projects[index].stack}</p>
+  <img src="./images/Counter.svg" alt="counter">
+  <p class="grey">${projects[index].year}</p>
+  `;
+  liveBtn.href = projects[index].liveVersionLink;
+  sourceBtn.href = projects[index].SourceLink;
+};
```

Comment:
- [ ] Good job implementing the changes requested by the previous CR 👏 , I noticed that the mobile version design does not stick to the Figma design in terms of margin and padding,  Kindly fix it to go to the next step 👍 

| Your design | Figma desing |
| :---: | :---: |
| ![Capture](https://github.com/SnakyMz/Portfolio/assets/80458199/cd3b029c-f6a0-4a32-9483-4ae33e2a2b90) | ![247774279-76b34d51-f07d-4392-a41f-477df90f63e5](https://github.com/SnakyMz/Portfolio/assets/80458199/d5f6cbff-ccba-416c-a0e4-49eb3b1a4a61) |

Reply:
Thanks for the review, will make changes as suggested.